cli-tests/scripts/dts-import.js

```diff
@@ -0,0 +1,50 @@
+import { resolve } from 'path';
+import { CUSTOM_TRANSFER_TOKEN_ACCESS_KEY } from '../constants';
+
+const {
+  file: {
+    providers: { createLocalFileSourceProvider },
+  },
+  strapi: {
+    providers: { createRemoteStrapiDestinationProvider },
+  },
+  engine: { createTransferEngine },
+} = require('@strapi/data-transfer');
+
+/**
+ * Reset the DB and import data from a DTS dataset
+ */
+export const resetDatabaseAndImportDataFromPath = async (filePath) => {
+  const source = createSourceProvider(filePath);
+  const destination = createDestinationProvider();
+
+  const engine = createTransferEngine(source, destination, {
+    versionStrategy: 'ignore',
+    schemaStrategy: 'ignore',
+  });
+
+  engine.diagnostics.onDiagnostic(console.log);
+
+  try {
+    await engine.transfer();
+  } catch (e) {
+    console.error('Import process failed.');
+    console.error(e);
+    process.exit(1);
+  }
+};
+
+const createSourceProvider = (filePath) =>
+  createLocalFileSourceProvider({
+    file: { path: resolve(filePath) },
+    encryption: { enabled: false },
+    compression: { enabled: false },
+  });
+
+const createDestinationProvider = () => {
+  return createRemoteStrapiDestinationProvider({
```

Comment:
Do we have access to the Strapi instances from the tests?

Reply:
The test runner provides the path of the available test apps, so we should be able to use that to load it as needed for the import (just like the import cli command) rather than attempting to keep it running in a separate process

Then we can face the issue of keeping it running in a separate process for the remote provider tests later, rather than depending on solving that right now before writing any tests

Reply:
> The test runner provides the path of the available test apps, so we should be able to use that to load it as needed for the import (just like the import cli command) rather than attempting to keep it running in a separate process

Alright, for now I was just re-using what we have for the e2e tests, but if we have a way of using the local provider instead of the remote one, I'm in :slightly_smiling_face: 

> Then we can face the issue of keeping it running in a separate process for the remote provider tests later, rather than depending on solving that right now before writing any tests

Not sure I understood this one

Reply:
Ohh, wait, this somehow works with playwright already just for data seeding? I thought you were anticipating having the server up and available on a specific the entire time we had the tests running.

If this works as-is, might as well leave it, I misunderstood.

Reply:
> Ohh, wait, this somehow works with playwright already just for data seeding?

Yep https://github.com/strapi/strapi/blob/cli-tests/seed-capabilities/e2e/scripts/dts-import.js#L67